app/src/main/java/org/mozilla/rocket/content/travel/data/TravelDataSource.kt

```diff
@@ -72,19 +72,4 @@ data class Video(
     val viewCount: Int,
     val date: String,
     val linkUrl: String
-)
-
-data class Hotel(
```

Comment:
I know @hartsu should review the PR first. Just a quick response that we should keep this `Hotel` class as UI model, and use data mapper to convert the API data (`BcHotelApiEntity`) to UI model in view model, like what we did in Game/Shopping.

Reply:
@benjamin-cheng There is already a class `HotelUiModel` being used as the UI model for hotel. TravelMapper is now converting the API data (`BcHotelApiEntity`) to UI model (`HotelUiModel`) in view model like Game/Shopping. This looks good to me.

Reply:
OK, somehow I misunderstand that the UI model was removed in this PR. It is an invalid concern.